lib/SILGen/ResultPlan.cpp

```diff
@@ -596,6 +597,10 @@ class ForeignAsyncInitializationPlan final : public ResultPlan {
     return ManagedValue::forUnmanaged(block);
   }
 
+  void deferExecutorBreadcrumb(ExecutorBreadcrumb &&breadcrumb) override {
+    this->breadcrumb = breadcrumb;
```

Comment:
Should this also be `std::move(breadcrumb)`?

Reply:
Probably should be. I also think it would be good to add a defensive assert to ensure an existing breadcrumb isn't being overwritten, since I only expect one crumb to ever be deferred. I'll make a follow-up PR on `main` for this. Shouldn't matter for 5.7

Reply:
taking care of that in https://github.com/apple/swift/pull/58649